src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -168,10 +781,372 @@ - (NSRect)accessibilityFrame
 
 - (id _Nullable)accessibilityParent
 {
-    return [self accessibilityParentAttribute];
+    return NSAccessibilityUnignoredAncestor([self parent]);
+}
+
+- (BOOL)isAccessibilityEnabled
+{
+    JNIEnv* env = [ThreadUtilities getJNIEnv];
+    GET_CACCESSIBILITY_CLASS_RETURN(NO);
+    DECLARE_STATIC_METHOD_RETURN(jm_isEnabled, sjc_CAccessibility, "isEnabled", "(Ljavax/accessibility/Accessible;Ljava/awt/Component;)Z", NO);
+
+    BOOL value = (*env)->CallStaticBooleanMethod(env, sjc_CAccessibility, jm_isEnabled, fAccessible, fComponent);
+    CHECK_EXCEPTION();
+    if (!value) {
+        NSLog(@"WARNING: %s called on component that has no accessible component: %@", __FUNCTION__, self);
```

Comment:
I see this warning being thrown when trying to use the disabled slider. If we run SwingSet2 and go to Slider demo. Try to navigate to disabled sliders using the VO keys. This warning is thrown when the VO focus is shifted to disabled sliders. I don't see similar warning before this change. This is the warning I see

" WARNING: -[CommonComponentAccessibility isAccessibilityEnabled] called on component that has no accessible component: <SliderAccessibility: 0x7fc938147c00>"

Reply:
Done